packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -136,11 +141,12 @@ SimpleForm.defaultProps = {
 
 const enhance = compose(
     connect((state, props) => ({
+        form: props.form || REDUX_FORM_NAME,
```

Comment:
why is it in `connect` while it doesn't depend on `state`?

Reply:
it should not depend on `state`, we're setting the redux-form name here. I tried to set `defaultProps` on enhance but it didn't work. This achieve the same results by setting the `form` prop **before** the `reduxForm` HOC is applied

Reply:
I could have used the `withProps` HOC from recompose though but thought it was unnecessary to introduce yet another one just for that